modules/oadp-installing-dpa-1-2-and-earlier.adoc

```diff
@@ -373,3 +375,29 @@ NAME                                                           DESIRED   CURRENT
 replicaset.apps/oadp-operator-controller-manager-67d9494d47    1         1         1       2m9s
 replicaset.apps/velero-588db7f655                              1         1         1       96s
 ----
+
```

Comment:
@skrthomas 

I was asked yesterday to add a Verification section to this Module.
Would you be able to do a peer-review on this new section or do I need to label it and request a peer-review?
Thank you for your help!
Carmi